data/contacts.metainfo.xml.in

```diff
@@ -1,35 +1,35 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2018 elementary, Inc. <contact@elementary.io> -->
 <component type="desktop-application">
-  <id>io.elementary.friends</id>
-  <launchable type="desktop-id">io.elementary.friends.desktop</launchable>
-  <translation type="gettext">io.elementary.friends</translation>
+  <id>io.elementary.contacts</id>
+  <launchable type="desktop-id">io.elementary.contacts.desktop</launchable>
+  <translation type="gettext">io.elementary.contacts</translation>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-3.0+</project_license>
 
-  <name>Friends</name>
+  <name>Contacts</name>
   <summary>Find and connect with contacts</summary>
   <description>
-    <p>See and contact your friends from elementary OS</p>
```

Comment:
I think this requires a rephrasing. IMO "contact your contact" sounds odd

Reply:
You're right. What about `get in touch with your contacts`?

Reply:
Fixed in 26e056faa47e63fcb0bb3bb6b25c7744735f4314